pontoon/base/static/css/translate.css

```diff
@@ -1352,6 +1352,15 @@ p#sign-in-required > a#sidebar-signin {
   top: 43px;
 }
 
+#helpers > section.other-locales {
+  padding-top: 5px;
+}
```

Comment:
We should keep the current UI unchanged for non-authenticated users, which this will break.